### PR TITLE
Added effects for clown and mime food.

### PR DESCRIPTION
**Pull request by Frutzl**
Made it so that the Clownburger pumps a little bit of honk serum into the
consumer, just like the honk slider.

Added silencing effects to Silencer, akin to Neurotoxin.

Added a small amount of Silencer for mime food.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -3790,6 +3793,11 @@
 	desc = "..."
 	icon_state = "slider_mime"
 
+/obj/item/weapon/reagent_containers/food/snacks/slider/mime/New()
+	..()
+	reagents.add_reagent(SILENCER, 2.5)
+	bitesize = 2.5"
```
Comment (Exxion) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: `"`

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -4895,6 +4895,12 @@
 	nutriment_factor = FOOD_METABOLISM
 	color = "#664300" //rgb: 102, 67, 0
 
+/datum/reagent/drink/silencer/on_mob_life(var/mob/living/M)
+
+ 	if(..())
+            	 return 1
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm]: The problem is indentation on this line and the next one, but I'm not actually sure what's wrong with it. Might you have indented with spaces? That's what it looks like on this page but I'm not sure if that's just a GitHub thing or what, or if BYOND even cares.

Comment (Frutzl) [reply code/modules/reagents/Chemistry-Reagents.dm]: yeah, that seems to be it
somehow i used spaces instead of tabs, how embarrasing

**Comment by clusterfack** [comment]
Neat

**Pull request merged**